x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_count_panel/index.tsx

```diff
@@ -40,7 +39,7 @@ export const AlertsCountPanel = memo<AlertsCountPanelProps>(
     // create a unique, but stable (across re-renders) query id
     const uniqueQueryId = useMemo(() => `${DETECTIONS_ALERTS_COUNT_ID}-${uuid.v4()}`, []);
     const [selectedStackByOption, setSelectedStackByOption] =
-      useState<AlertsStackByField>(DEFAULT_STACK_BY_FIELD);
+      useState<string>(DEFAULT_STACK_BY_FIELD);
```

Comment:
```suggestion
      useState(DEFAULT_STACK_BY_FIELD);
```

💅 TS will automatically assume it's a string